client/grpc/util/util.go

```diff
@@ -0,0 +1,21 @@
+package util
+
+import (
+	"github.com/kava-labs/kava/app"
+	"github.com/kava-labs/kava/app/params"
+	query "github.com/kava-labs/kava/client/grpc/query"
+)
+
+// Util contains utility functions for the Kava gRPC client
+type Util struct {
+	query          *query.QueryClient
+	encodingConfig params.EncodingConfig
+}
+
+// NewUtil creates a new Util instance
+func NewUtil(query *query.QueryClient) *Util {
+	return &Util{
```

Comment:
hmm now I'm curious as to if we even need this util package and type - could be simpler to have the top level client use the query clients already associated with versus constructing this other client to make calls to the same api (e.g. the query client from

```

// KavaGrpcClient enables the usage of kava grpc query clients and query utils
type KavaGrpcClient struct {
	config KavaGrpcClientConfig

	// Query clients for cosmos and kava modules
	Query *query.QueryClient
```
already has access to the account client